scss/footer.scss

```diff
@@ -0,0 +1,24 @@
+.footer {
+  align-items: center;
+  background-color: $blue;
+  display: flex;
+  flex-direction: column;
+  padding: 30px;
+  position: relative;
+
+  .footer-logo {
+    position: relative;
+    margin-top: 20px;
+  }
+
+  .footer-copyright {
+    color: $white;
+    font-size: $text-small;
+    align-self: flex-end;
+  }
+
+  .footer-text {
```

Comment:
you need a margin bottom here too, to separate this from the image. it's always preferred to use bottom instead of top and right instead of left because we encounter the leftmost and topmost elements first.